container/src/LuigiCompoundContainer.svelte

```diff
@@ -1,6 +1,6 @@
 <svelte:options
   customElement={{
-    tag: null,
+    tag: '',
```

Comment:
Changed because of reported 'svelte_options_invalid_tagname' error:
https://svelte.dev/docs/svelte/compiler-errors#svelte_options_invalid_tagname